stdlib/planner/group_count_eval_test.flux

```diff
@@ -0,0 +1,55 @@
+package planner_test
+
+import "testing"
+import "planner"
+
+option planner.disablePhysicalRules = ["PushDownWindowAggregateRule"]
+
+input = "
+#datatype,string,long,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,true,true,true,false
+#default,_result,,,,,,
+,result,table,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,system,hostA,load1,1.83
+,,0,2018-05-22T19:53:36Z,system,hostA,load1,1.72
+,,0,2018-05-22T19:53:37Z,system,hostA,load1,1.77
+,,0,2018-05-22T19:53:56Z,system,hostA,load1,1.63
+,,0,2018-05-22T19:54:06Z,system,hostA,load1,1.91
+,,0,2018-05-22T19:54:16Z,system,hostA,load1,1.84
+
+,,1,2018-05-22T19:53:26Z,system,hostB,load3,1.98
+,,1,2018-05-22T19:53:36Z,system,hostB,load3,1.97
+,,1,2018-05-22T19:53:46Z,system,hostB,load3,1.97
+,,1,2018-05-22T19:53:56Z,system,hostB,load3,1.96
+,,1,2018-05-22T19:54:06Z,system,hostB,load3,1.98
+,,1,2018-05-22T19:54:16Z,system,hostB,load3,1.97
+
+,,2,2018-05-22T19:53:26Z,system,hostC,load5,1.95
+,,2,2018-05-22T19:53:36Z,system,hostC,load5,1.92
+,,2,2018-05-22T19:53:41Z,system,hostC,load5,1.91
+,,2,2018-05-22T19:53:46Z,system,hostC,load5,1.92
+,,2,2018-05-22T19:53:56Z,system,hostC,load5,1.89
+,,2,2018-05-22T19:54:16Z,system,hostC,load5,1.93
```

Comment:
It would be good to change up the tags across the same host here, to show that the grouping+aggregating is happening correctly.  For example, you could change the last three of these rows above to `load1` (and also bump the table number).

Reply:
(ditto for the other _test.flux files)

Reply:
@wolffcm made these suggested changes, thanks